src/screens/account/components/view-key/view-key.tsx

```diff
@@ -32,6 +33,12 @@ export class ViewKeyComponent extends React.Component<
         this.state = {
             copied: false
         };
+
+        forbidFunction();
```

Comment:
well, this is weird :) forbidFunction could mean anything in this context :)
but using `forbidScreenshot` it's more self explanatory